pallet/src/assets/move-projects/smove-build-all.sh

```diff
@@ -18,13 +18,25 @@ bundle_dir=(
 )
 
 # Build simple packages
-for i in "${build_dir[@]}"; do
-    echo $i
-    smove build -p $i
+for dir in "${build_dir[@]}"; do
+    echo $dir
+    build_script=$dir"/build.sh"
+    if [ -f "$build_script" ];
+    then
+        sh $build_script
+    else
+        smove build -p $dir
+    fi
 done
 
 # Build bundles
-for i in "${bundle_dir[@]}"; do
-    echo $i
-    smove bundle -p $i
+for dir in "${bundle_dir[@]}"; do
+    echo $dir
+    build_script=$dir"/build.sh"
+    if [ -f "$build_script" ];
+    then
+        sh $build_script
+    else
+        smove bundle -p $dir
```

Comment:
We would need to assume the `build.sh` contains the `smove bundle` command.

I dunno, maybe we can keep the original script snippet for bundles to avoid confusion.
Up to you to decide 😄